modules/environment-roles/templates/terraform_scripts_policy.json.tpl

```diff
@@ -62,23 +62,40 @@
         "logs:ListTagsLogGroup",
         "logs:PutRetentionPolicy",
         "logs:TagLogGroup",
+        "rds:DescribeDBClusters",
+        "rds:ListTagsForResource",
         "s3:GetObject",
         "s3:PutObject",
-        "ssm:GetParameter"
+        "ssm:CreateDocument",
+        "ssm:DeleteDocument",
+        "ssm:DescribeDocument",
+        "ssm:DescribeDocumentPermission",
+        "ssm:GetDocument",
+        "ssm:GetParameter",
+        "ssm:UpdateDocument",
+        "ssm:UpdateDocumentDefaultVersion"
       ],
       "Resource": [
         "arn:aws:dynamodb:eu-west-2:${account_id}:table/tdr-terraform-state-lock-scripts",
         "arn:aws:iam::${account_id}:instance-profile/bastion",
         "arn:aws:iam::${account_id}:role/BastionEC2Role${title_environment}",
+        "arn:aws:iam::${account_id}:role/TDRBastionAccessDbRole${title_environment}",
+        "arn:aws:iam::${account_id}:policy/TDRBastionAccessDbPolicy${title_environment}",
+        "arn:aws:iam::${account_id}:policy/TDRBastionAssumeDbRolePolicy${title_environment}",
+        "arn:aws:iam::${account_id}:role/TDRJenkinsRunDocumentRoleIntg",
```

Comment:
Should "Intg" be hardcoded to the the `TDRJenkinsRunDocumentRoleIntg` and `TDRJenkinsRunDocumentPolicyIntg` resource names?

Reply:
Ah good spot, I thought I'd got all the hard coded ones. I've changed it.